_includes/domain-search.html

```diff
@@ -43,9 +43,10 @@
                     <div class="usa-alert__body">
                         <p class="usa-alert__text">
                             <b>${requested_domain_display_string}</b> isn't available.
-                            <a class="usa-link" href="{{ '/domains/choosing/' | url }}">
-                                Read more about choosing your .gov domain.
-                            </a>
+                            ${response.message} 
+                        <a class="usa-link" href="{{ '/domains/choosing/' | url }}">
```

Comment:
also a small formatting nit: I think the <a> tags should be indented to be aligned with line 46!